python/ray/scripts/scripts.py

```diff
@@ -577,6 +577,27 @@ def get_head_ip(cluster_config_file, cluster_name):
     click.echo(get_head_node_ip(cluster_config_file, cluster_name))
 
 
+@cli.command()
+def stack():
+    COMMAND = """
+pyspy=`which py-spy`
+if [ ! -e "$pyspy" ]; then
+    echo "ERROR: Please 'pip install py-spy' first"
+    exit 1
+fi
+export IFS="
+"
+sudo true
+for worker in `ps aux | grep default_worker.py | grep -v grep`; do
```

Comment:
This will find the raylet also because we pass the command for starting the worker as a command line argument into the raylet (however, printing an extra thing may not be an issue).